File: 180525/codefight/arcade/intro/JungDongHyun/51. deleteDigit.py
def deleteDigit(n):
    n = str(n)
    maximum = 0
    for idx_to_remove in range(len(n)):
        modified_string = n[0:idx_to_remove] + n[idx_to_remove + 1:len(n)]
        result = int(modified_string)
        maximum = max(result, maximum)
    
    return maximum

File: 180525/codefight/arcade/intro/NaDongHyun/51.deleteDigit.js
function deleteDigit(n) {
  var range = [];
  for(let i = 0; i < (n+"").length; i ++) {
      var string = (n+"");
      if(i !== string.length) {
          range.push(parseInt(string.substring(0,i) + string.substring(i+1, string.length), 10));
      }
  }
  return Math.max(...range);
}
